課題①‐３/main.js

```diff
@@ -0,0 +1,30 @@
+"use strict"
+
+function show() {
```

Comment:
是非アロー関数を利用しましょう！
あと `show` という名前だと「何を Show する」のかが分からないので
`showTasks` という名前はいかがでしょうか？

```suggestion
const show = () => {
```